cli/src/generate/grammar_files.rs

```diff
@@ -213,6 +213,18 @@ pub fn generate_grammar_files(
                     updated = true;
                 }
 
+                // insert `tree-sitter` right after `files`
+                if !package_json.contains_key("tree-sitter") {
+                    eprintln!("Adding tree-sitter to package.json");
+                    package_json = insert_after(
+                        package_json,
+                        "files",
+                        "tree-sitter",
+                        json!([{"file-types": language_name}]),
```

Comment:
`file-types` should be an array

```suggestion
                        json!([{"file-types": [language_name]]),
```

also, since this should always be at the end, we could just use `package_json.insert` in this case

```rs
package_json.insert(
    "tree-sitter".to_string(),
    json!([{"file-types": [language_name]}]),
);
```